src/utils.c

```diff
@@ -1886,7 +1886,7 @@ GSList *utils_get_config_files(const gchar *subdir)
 
 /* Suffix can be NULL or a string which should be appended to the Help URL like
  * an anchor link, e.g. "#some_anchor". */
-gchar *utils_get_help_url(const gchar *suffix)
+static gchar *utils_get_help_url(const gchar *suffix)
```

Comment:
Since this is no longer used anywhere else, why no just add `utils_open_browser()` at the end and make this `utils_open_help()` instead of adding another bit of spaghetti?